core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -229,4 +231,26 @@ public Duration getTenantIdleTimeout() {
     public void setTenantIdleTimeout(final Duration tenantIdleTimeout) {
         this.tenantIdleTimeout = Objects.requireNonNull(tenantIdleTimeout);
     }
+
+    /**
+     * Sets the configured mappers for this adapter
+     * <p>
+     * Setting this property to an empty hashMap will disable mapping for this adapter.
+     *
+     * @param mapperEndpoints The new hashMap with mapper endpoints
+     * @throws NullPointerException if parameter is {@code null}.
+     */
+    public final void setMapperEndpoints(final HashMap<String, MapperEndpoint> mapperEndpoints) {
+        this.mapperEndpoints = Objects.requireNonNull(mapperEndpoints);
```

Comment:
you should better copy all elements, i.e.
```
this.mapperEndpoints.clear();
this.mapperEndpoints.addAll(mapperEndpoints);
```
so that future modifications of the passed in map have no impact on this config class